From cd50917675d5de3df034c4faae5d8b245ef27a9b Mon Sep 17 00:00:00 2001
From: John Cox <jc@kynesim.co.uk>
Date: Thu, 27 Jun 2024 12:53:46 +0100
Subject: [PATCH 13/17] v4l2_stateful: Add info line to indicate V4L2 decode

Very helpful for testing (automated & otherwise) to show we're actually
trying to use h/w decode.
---
 media/gpu/v4l2/v4l2_video_decoder_backend_stateful.cc | 1 +
 1 file changed, 1 insertion(+)

--- a/src/media/gpu/v4l2/v4l2_video_decoder_backend_stateful.cc
+++ b/src/media/gpu/v4l2/v4l2_video_decoder_backend_stateful.cc
@@ -131,6 +131,7 @@ bool V4L2StatefulVideoDecoderBackend::In
   framerate_control_ = std::make_unique<V4L2FrameRateControl>(
       base::BindRepeating(&V4L2Device::Ioctl, device_), task_runner_);
 
+  LOG(INFO) << "V4L2StatefulVideoDecoderBackend: Initialized OK";
   return true;
 }
 
